.github/workflows/require-allow-edits.yml

```
@@ -0,0 +1,14 @@
+name: Require “Allow Edits”
+
+on: [pull_request]
+
+jobs:
+  _:
+    name: "Require “Allow Edits”"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: ljharb/require-allow-edits@main
```

```suggestion
    - uses: ljharb/require-allow-edits@a200d61
```
unless you're ok with having this be a floating reference